docfx/en/components/toc.json

```diff
@@ -63,34 +63,28 @@
     "href": "general-licensing.md",
     "status": ""
   },
+  {
+    "exclude": ["Blazor", "React", "WebComponents"],
+    "name": "Changelog",
+    "href": "general-changelog-dv.md",
+    "status": "UPDATED"
+  },
   {
     "exclude": ["Angular", "React", "WebComponents"],
-    "name": "How to",
-    "status": "NEW",
-    "items": [
-      {
-        "exclude": ["Angular", "React", "WebComponents"],
-        "name": "Blazor Excel Library",
-        "href": "blazor-excel-library-temporarily-stop-automatic-calculation-of-formulas-to-speed-up-processing.md",
-        "status": ""
-      },
-      {
-        "exclude": ["Angular", "React", "WebComponents"],
-        "name": "How To Read & Write Excel Files To Reduce Server Load",
-        "href": "blazor-webassembly-how-to-read-and-write-excel-files-to-reduce-server-load.md",
-        "status": ""
-      },
-      {
-        "exclude": ["Angular", "React", "WebComponents", "Blazor"],
-        "name": "How To Customize Marker Display in Blazor Charts",
-        "href": "customize-marker-display-in-blazor-charts.md",
-        "status": ""
-      }
-    ]
```

Comment:
undo removal of these 3 topics
![image](https://user-images.githubusercontent.com/11665978/196533928-f6558417-ec67-479c-b444-857cd93b42f5.png)

Reply:
see 
https://github.com/IgniteUI/igniteui-xplat-docs/pull/383/commits/6d6fce67106c04140fb7dae241e45beadabc6d62